index.bs

```diff
@@ -375,7 +375,81 @@ request they are associated with.
 Authentication {#authentication}
 ================================
 
-Issue(102): Incorporate material from the \[J-PAKE](j-pake.md) document.
+In order for one agent (the challenger) to authenticate another (the responder),
+the challenger may send an authentication-request message and expect an
+authentication-response message to best sent back from the responder.  To
+mutually authenticate, this mechanism is used twice, once by each side acting as
+the challenger.  This mechanism assumes the agents share a low-entry secret,
+such as a number or a short password that could be entered by a user on a
+keyboard or TV remote control.
+
+For all messages and objects defined in this section, see Appendix A for the full
+CDDL definitions.
+
+The challenger sends an authentication-request message with the following values:
+
+-   mechanism: The authentication mechanism being used.  This standard only
+    defines the mechanism hkdf-of-scrypt-of-psk but this field gives a place for
+    other mechanisms to be specified.
+
+-   salt: 32 random bytes (which need not be kept secret). 
```

Comment:
Define suitable method for generating these bytes?  Presumably the [TLS 1.3](https://tools.ietf.org/html/rfc8446#appendix-C.1) suggestions are appropriate here.

Reply:
fixed

Reply:
Sorry, not fixed.  Wrong comment

Reply:
It's for HKDF, not TLS, so the properties and details are at https://tools.ietf.org/html/rfc5869#section-3.1.  But that's a good point that we should point to that section, so I added a link to it.